dotcom-rendering/src/web/components/FrontCard.tsx

```diff
@@ -4,7 +4,26 @@ import { Card } from './Card/Card';
 
 type Props = {
 	trail: TrailType;
-} & Partial<CardProps>;
+	headlineSize?: SmallHeadlineSize;
+	showQuotes?: boolean;
+	showByline?: boolean;
+	imageUrl?: string;
+	imagePosition?: ImagePositionType;
+	imagePositionOnMobile?: ImagePositionType;
+	imageSize?: ImageSizeType;
+	trailText?: string;
+	avatar?: AvatarType;
+	kickerText?: string;
+	showPulsingDot?: boolean;
+	showSlash?: boolean;
```

Comment:
I think the defaults should be rules for these. I don't think we have any use cases where we would want to override these rules.